### DISCUSSION TITLE
Aarch64 support

### DISCUSSION THREAD

**Discussion by andrii0lomakin** [oneapi-src/oneDAL#2892] [Q&A] [answered]
Good day.
I have noticed that only x86 processors are listed in system requirements.
Is it possible to use oneDAL library in aarch64 ISA ?

By "possible" I mean both the project compilation using the aarch64 target and the ability to achieve performance gains.

**Comment by napetrov** [comment, marked as answer]
@andrii0lomakin  - yes ARM supported. you can check on ci process for building ARM - https://github.com/oneapi-src/oneDAL/blob/main/.ci/pipeline/ci.yml#L98

This is work in progress but in general things are working on ARM and provide acceleration
